practice-two/src/javascript/modules/filterNotes.js

```diff
@@ -0,0 +1,42 @@
+import httpUtils from '../utils/httpUtils';
+import { renderNote, renderAllNotes } from './getNotes'; 
+
+async function renderNotesByCategory(category) {
```

Comment:
@quanghuy1799 You are applying ES6 for the practice. Please use the arrow function instead. Make sure all code is consistent.